src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -2043,8 +2045,9 @@ def test_azconfig_aad_auth(self, resource_group, location):
             'endpoint': endpoint
         })
 
-        # Before assigning data reader role, read operation should fail with AAD auth
-        with self.assertRaisesRegex(CLIError, "Operation returned an invalid status 'Unauthorized'"):
+        # Before assigning data reader role, read operation should fail with AAD auth.
+        # The exception really depends on the which identity is used to run this testcase.
+        with self.assertRaisesRegex(CLIError, "Operation returned an invalid status '(?:Unauthorized|Forbidden)'"):
```

Comment:
There was a bug ( [fixed now](https://github.com/Azure/azure-cli/pull/15698) ) in `get_login_credentials()` functionality because of which @fengzhou-msft must have gotten Unauthorized error when he updated the test [here](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-cli%2Fcommit%2Fa3dd306d8a66cbf0e930815013bcd44d39ad18df%23diff-cb948504612e42a7f692f8aa2123d78b9e0d5753b970631097410962c737e7a3&data=04%7C01%7CAvani.Gupta%40microsoft.com%7Cf7448ee5e8d44a5b50c808d8847eee82%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637405028002353126%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=daYCB%2Bu0ymZAmxqhzXzZ7EKy4ITyttOuKcNE7h53m3M%3D&reserved=0).


We should always get 'Forbidden' error if the `get_login_credentials()` functionality returns the correct access token from AAD.

Reply:
Will fix in a new pr.